python/python-copr.spec

```diff
@@ -137,7 +137,7 @@ Requires: python3-future
 
 %{?python_provide:%python_provide python3-copr}
 
-%if 0%{?fedora} > 30 || 0%{?rhel} > 8
+%if 0%{?fedora} > 30 || 0%{?rhel} > 8 || 0%{?openEuler}
```

Comment:
This seems like an ideal candidate for '%else' branch above to `%if 0%{?rhel} && 0%{?rhel} <= 8`

Reply:
> This seems like an ideal candidate for '%else' branch above to `%if 0%{?rhel} && 0%{?rhel} <= 8`

Done/